inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_nms_5_to_legacy.cpp

```diff
@@ -102,18 +102,21 @@ ngraph::pass::ConvertNMS5ToLegacyMatcher::ConvertNMS5ToLegacyMatcher() {
         Output<Node> output_0 = nms_legacy->output(0);
         if (nms_5->output(0).get_element_type() != output_0.get_element_type()) {
             output_0 = std::make_shared<opset1::Convert>(output_0, nms_5->output(0).get_element_type());
-            output_0.get_node_shared_ptr()->set_friendly_name(nms_5->get_friendly_name() + "/convert.0");
```

Comment:
PR description and the fix are different 😄 

Looks like you just changed the names of convert ops.
Can we add a test to check that it solve the issue?

Reply:
> PR description and the fix are different 😄

Yea, thanks - I forgot to update description after I updated changes.

> Can we add a test to check that it solve the issue?

Not sure, since it's about network tests which are not a part of OpenVINO repo as I understand. Anyway, @GlebKazantaev is looking into this to prepare another fix version